jupyter_book/toc.py

```diff
@@ -41,6 +41,9 @@ def add_toctree(app, docname, source):
     if not app.config["globaltoc_path"]:
         return
 
+    # Some behavior we only trigger with an HTML build
+    is_html = "latex" not in app.builder.name
```

Comment:
@choldgraf not sure why this is coming as a diff after merging in `latest master`